composer.json

```diff
@@ -13,7 +13,7 @@
     ],
     "license": "MIT",
     "require": {
-        "laravel/framework": "5.0.*|5.1.*|5.2.*|5.3.*|5.4.*"
+        "laravel/framework": "5.0.*|5.1.*|5.2.*|5.3.*|5.4.*|5.5.*"
```

Comment:
@mattstauffer Good call, I removed the 5.5 upgrade and just left the package autodiscovery for this PR.